### PR TITLE
UCS/DEBUG: Support UCX_HANDLE_ERRORS=none

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Review by hoopoepg — APPROVED** [review]

File: src/ucs/debug/debug.c [hunk 1]
```diff
@@ -1080,12 +1080,20 @@ void ucs_handle_error(const char *message)
     }
 }
 
+int ucs_debug_is_handle_errors()
+{
+    static const unsigned mask = UCS_BIT(UCS_HANDLE_ERROR_BACKTRACE) |
+                                 UCS_BIT(UCS_HANDLE_ERROR_FREEZE) |
+                                 UCS_BIT(UCS_HANDLE_ERROR_DEBUG);
+    return !!(ucs_global_opts.handle_errors & mask);
```
Comment (brminich) [inline src/ucs/debug/debug.c:1088]: why need `!!` here? Typically we do not do it for booleans, just for the cases where `0` or `1` is needed

Comment (yosefe) [reply src/ucs/debug/debug.c]: @brminich fixed

**Comment by yosefe** [comment]
@brminich can you pls look?

**Comment by brminich** [comment]
should we also update description in `ucs_global_opts_table` and mention that none is exclusive with other modes?

**Review by hoopoepg — APPROVED** [review]

**Review by brminich — APPROVED** [review]

**Pull request merged**